web/package-lock.json

```diff
@@ -51,9 +51,9 @@
       "license": "MIT"
     },
     "node_modules/@auth0/auth0-vue": {
-      "version": "2.3.3",
```

Comment:
@burkkyy There are more changes than I'm expecting in the lock file.
I'd only expect vuetify related updates.